rhymes.py

```diff
@@ -142,11 +144,11 @@ def humanize_stress(stress):
     :param stress: CMU-defined stress marker (1 > 2 > 0)
     :return converted_stress: ordered stress marker
     '''
-    if stress is 1:
+    if stress == 1:
         return 2
-    if stress is 2:
+    if stress == 2:
         return 1
-    if stress is None:
+    if stress == None:
```

Comment:
`if stress is None` is the canonical way to put this. (there is only one `None`...)

In fact, the linter will complain about `x == None`